lib/ss_light.go

```diff
@@ -301,6 +323,9 @@ func (l *LightClient) Start(
 				case <-ctx.Done():
 					return
 				case <-time.After(time.Second * 30):
+					// STEP : Re-fetching metadata
+					showStep(success, "Re-fetching metadata", l.jsonOut)
+
 					// Allow 30 seconds to see if new leaders were added
 					newMtdt, err := getInfo(sharable, metadata.Cookie.Id, l.pubKey)
```

Comment:
Lets remove the refetching now. This was required due to the previous flow.... 

Now we should just retry connecting to the same list if count < len(metadata.Cookie.Leaders)